byway/src/main/webapp/destinations.js

```diff
@@ -218,21 +237,50 @@ function getCurrentAddress(){
 * add event listener for submit button
 */
 window.onload = function(){
+  
   let nextButton = document.getElementById('next-button');
   nextButton.addEventListener('click', () => {
-      nextButton.href = buildUrlWithParams("/interests.html", {tripKey});
+      nextButton.href = configureTripKeyForPath(tripKey, '/interests.html');
   });
   document.getElementById('user-input-form').addEventListener('submit', (event) => {
     event.preventDefault();
-    const formData = new  FormData(document.getElementById("user-input-form"));
-    fetch(buildUrlWithParams("/api/destinations", {tripKey}), {method: 'POST', body:formData}).then((response)=>
-        response.json()).then(locationData => {
-        updateLocations(locationData);
-        updateStartDestination(locationData);
-      });
-    });
+    sendPlaceId("destinations-search-box");
+    sendPlaceId("start-search-box");
```

Comment:
Can you combine these two calls so you don't have to make 2 separate requests to the server? Also would prevent you from updating the UI twice

Reply:
I've tried to combine them, but since it would have to call findPlacefromQuery two seperate times, I wasn't able to get it to work to update both the start and destination params at the same time

Reply:
I just updated it so that savePlaceId does both calls togeth